.samples.config.json

```diff
@@ -8,7 +8,6 @@
     },
     "server": {
       "staticDir": "dist",
-      "startSignal": "Express server started",
```

Comment:
Reminder when updating the generator to also remove the stuff about this in the readme.